### DISCUSSION TITLE
I want to ask something.

### DISCUSSION THREAD

**Discussion by EdudModnar** [MonoGame/MonoGame#8578] [General]
Hey, I'm new around here, and I want to download MonoGame engine, but which one do I need to download? One thing, or some of them or all of them? I'm confused about it. I'm on a Win10 operating system, and I need some help about it.

**Comment by EdudModnar** [comment]
Looks like I have to go back to unity then.

**Comment by Abyz08** [comment]
Follow the tutorial [here](https://docs.monogame.net/articles/getting_started/index.html)
It should have all of the support you need to get up and running!
